test/weth9.test.ts

```diff
@@ -1,28 +1,17 @@
 import {WETH9Instance} from '../build/types/truffle-types'
 const BigNumber = require('bignumber.js')
-const {expectEvent, expectRevert} = require('@openzeppelin/test-helpers')
+const {expectEvent} = require('@openzeppelin/test-helpers')
 const WETH9 = artifacts.require('WETH9.sol')
-// address(0)
-const ZERO_ADDR = '0x0000000000000000000000000000000000000000'
 
 contract('WETH9', ([sender]) => {
   let weth: WETH9Instance
 
-  const underlyingAsset = ZERO_ADDR
-
   before('Deployment', async () => {
     // deploy USDC token
     weth = await WETH9.new()
   })
 
   describe('Send ETH', () => {
-    /*it('should revert whitelisting a product from non-owner address', async () => {
-        await expectRevert(
-            whitelist.whitelistProduct(underlyingAsset, usdc.address, usdc.address, {from: random}),
-            'Ownable: caller is not the owner',
-        )
-        })*/
-
```

Comment:
why did oyu comment this out?

Reply:
I think that test is copied from somewhere and pasted there... it have nothing to do with testing WETH right ? and I don't remember writing it there!